examples/api/.env.example

```diff
@@ -1,9 +1,13 @@
 HOST_URL=http://localhost:4000
 MONGO_URL=mongodb://localhost:27017/wepublish
+MONGO_LOCALE=en
 WEBSITE_URL=http://localhost:5000
+# PORT
```

Comment:
If you comment something which you can uncomment, you should not add extra space. 

Make it
```
#PORT=
```

Same goes for others.

Reply:
Space you usually add when you want to add a pernament comment e.g.
```
# Here is the beloved Stripe configuration
```